src/main/java/com/guigs44/farmingforengineers/client/render/RenderMerchant.java

```diff
@@ -1,47 +1,32 @@
-// package com.guigs44.farmingforengineers.client.render;
-//
-// import com.guigs44.farmingforengineers.FarmingForEngineers;
-// import com.guigs44.farmingforengineers.entity.EntityMerchant;
-// import net.minecraft.client.model.ModelVillager;
-// import net.minecraft.client.renderer.GlStateManager;
-// import net.minecraft.client.renderer.entity.RenderLiving;
-// import net.minecraft.client.renderer.entity.RenderManager;
-// import net.minecraft.client.renderer.entity.layers.LayerCustomHead;
-// import net.minecraft.util.ResourceLocation;
-//
-// public class RenderMerchant extends RenderLiving<EntityMerchant> {
-//
-// private static final ResourceLocation MERCHANT_TEXTURE = new ResourceLocation(FarmingForEngineers.MOD_ID,
-// "textures/entity/merchant.png");
-//
-// public RenderMerchant(RenderManager renderManager) {
-// super(renderManager, new ModelVillager(0f), 0.5f);
-// addLayer(new LayerCustomHead(this.getMainModel().villagerHead));
-// shadowSize = 0.5f;
-// }
-//
-// @Override
-// public ModelVillager getMainModel() {
-// return (ModelVillager) super.getMainModel();
-// }
-//
-// @Override
-// protected ResourceLocation getEntityTexture(EntityMerchant entity) {
-// return MERCHANT_TEXTURE;
-// }
-//
-// @Override
-// protected void preRenderCallback(EntityMerchant merchant, float partialTickTime) {
-// float scale = 0.9375f;
-// GlStateManager.scale(scale, scale, scale);
-// int diggingAnimation = merchant.getDiggingAnimation();
-// if(diggingAnimation > 0) {
-// GlStateManager.translate(0,diggingAnimation * 0.05, 0);
-// }
-// }
-//
-// @Override
-// protected boolean canRenderName(EntityMerchant entity) {
-// return entity.getDiggingAnimation() <= 0 && super.canRenderName(entity);
-// }
-// }
+package com.guigs44.farmingforengineers.client.render;
+
+import net.minecraft.client.model.ModelVillager;
+import net.minecraft.client.renderer.entity.RenderLiving;
+import net.minecraft.entity.Entity;
+import net.minecraft.util.ResourceLocation;
+
+import com.guigs44.farmingforengineers.FarmingForEngineers;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+/**
+ * A renderer for {@link com.guigs44.farmingforengineers.entity.EntityMerchant}. Just a re-skinned villager.
+ */
+@SideOnly(Side.CLIENT)
```

Comment:
`@SideOnly` is not needed here

Reply:
I'm never quite sure with these annotations...

Happy to remove it, though it seems like all subclasses of `RenderLiving` (`RenderCow`, `RenderVillager`, `RenderWither`, etc.) have explicit `SideOnly` annotations.